tests/testsuite/check.rs

```diff
@@ -772,3 +778,116 @@ fn error_from_deep_recursion() -> Result<(), fmt::Error> {
 
     Ok(())
 }
+
+#[cargo_test]
+fn workspace_rustc_wrapper_affects_all_workspace_members() {
+    let p = project()
+        .file(
+            "Cargo.toml",
+            r#"
+            [workspace]
+            members = ["bar", "baz"]
+        "#,
+        )
+        .file("bar/Cargo.toml", &basic_manifest("bar", "0.1.0"))
+        .file("bar/src/lib.rs", "pub fn bar() {}")
+        .file("baz/Cargo.toml", &basic_manifest("baz", "0.1.0"))
+        .file("baz/src/lib.rs", "pub fn baz() {}")
+        .build();
+
+    p.cargo("check")
+        .env("WORKSPACE_RUSTC_WRAPPER", "rustc-echo-wrapper")
+        .with_stdout_contains("WRAPPER CALLED: rustc --crate-name bar [..]")
+        .with_stdout_contains("WRAPPER CALLED: rustc --crate-name baz [..]")
+        .run();
+}
+
+#[cargo_test]
+fn workspace_rustc_wrapper_includes_path_deps() {
```

Comment:
Is this the expected behavior?

Reply:
Yes, because `path` dependencies are implicitly workspace members, I think it is consistent with the definition of what a "workspace" is.